src/two_player_collision_example.cpp

```diff
@@ -211,73 +199,73 @@ TwoPlayerCollisionExample::TwoPlayerCollisionExample(
       "NominalHeadingP2");
   // p2_cost.AddStateCost(p2_nominal_orientation_cost);
   
-  // Stay in lanes.
+  // cost for deviating from the center of the lane (for both p1 and p2)
   const Polyline2 lane1(
-      {Point2(6, -10.0), Point2(6, 10.0)});
-  const Polyline2 lane2(
-      {Point2(2.5, -1000.0), Point2(2.5, 1000.0)});
-  const Polyline2 lane3(
-      {Point2(-1, -10.0), Point2(-1, 10.0)});
-
-  const std::shared_ptr<QuadraticPolyline2Cost> p1_lane_cost(
+      {Point2(2.5, -50.0), Point2(2.5, 50.0)});
+  const std::shared_ptr<QuadraticPolyline2Cost> p1_lane1_cost(
```

Comment:
I'm not sure this is going to compile, unless maybe the GitHub interface is not showing me everything. It looks like `lane1` is not defined.

Reply:
lane1 is defined above your previous comment. The GitHub interface has made it a bit confusing to see what was kept and what was removed...

Reply:
Do you understand my points though?

Reply:
your comments in general? yes, working on them!